task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -500,6 +508,20 @@ def _monitor_subprocess(self):
 
             self._send_heartbeat_if_needed()
 
+            self._handle_task_overtime_if_needed()
+
+    def _handle_task_overtime_if_needed(self):
+        """Handle termination of auxiliary processes if the task exceeds the configured success overtime."""
+        if self._terminal_state != TerminalTIState.SUCCESS:
+            return
+
+        now = datetime.now(tz=timezone.utc)
+        self._overtime = (now - (self._task_end_datetime or now)).total_seconds()
```

Comment:
This should probably be using `time.monotonic()` etc not datetime, as if the clocks goes back due to DST this will get confused.